lib/import/hmcts_importer.rb

```diff
@@ -0,0 +1,105 @@
+module Import
+  class HmctsImporter
+    PUBLICATION_TYPE_SLUGS = {
+      "Form" => "forms",
+      "Guidance" => "guidance",
+    }.freeze
+
+    def initialize(dry_run)
+      @dry_run = dry_run
+    end
+
+    def import(csv_path)
+      importer_user = User.find_by(name: "Automatic Data Importer")
+      raise "Could not find 'Automatic Data Importer' user" unless importer_user
+
+      HmctsCsvParser.publications(csv_path).each do |publication_data|
+        publication_ids = []
+
+        begin
+          publication = Publication.new
+          publication.publication_type = PublicationType.find_by_slug(publication_type_slug(publication_data[:publication_type]))
+          publication.title = publication_data[:title]
+          publication.summary = publication_data[:summary]
+          publication.body = publication_data[:body]
+          publication.topics = publication_data[:policy_areas].map { |policy_area| Topic.find_by(name: policy_area) }
```

Comment:
should we be using `find_by!` heer so we get an error if the topic can't be found?